.github/workflows/upgrade-statechannels-latest.yml

```
@@ -0,0 +1,51 @@
+name: Upgrade statechannels packages to @latest tag
+on:
+  schedule:
+    - cron: '0 8 * * *'
+  # enable users to manually trigger with workflow_dispatch
+  workflow_dispatch: {}
```

I don't think we want to run this every day?
```suggestion
on:
  # enable users to manually trigger with workflow_dispatch
  workflow_dispatch: {}
```

---

Of course ...